pkg/asset/installconfig/gcp/validation.go

```diff
@@ -448,7 +457,7 @@ func checkRecordSets(client API, ic *types.InstallConfig, zone *dns.ManagedZone,
 	preexistingRecords := sets.New[string](records...).Intersection(setOfReturnedRecords)
 
 	if preexistingRecords.Len() > 0 {
-		errMsg := fmt.Sprintf("record(s) %q already exists in DNS Zone (%s/%s) and might be in use by another cluster, please remove it to continue", sets.List(preexistingRecords), ic.GCP.ProjectID, zone.Name)
+		errMsg := fmt.Sprintf("record(s) %q already exists in DNS Zone (%s/%s) and might be in use by another cluster, please remove it to continue", sets.List(preexistingRecords), project, zone.Name)
```

Comment:
I understand that we are modifying pre-existing code here, but do we really need to list the pre-existing records in this error message? Would it be sufficient to just report the length?

Reply:
I will note tis, but for the sake of time I think that leaving this as is should be ok.